README.md

```diff
@@ -26,33 +26,33 @@ This integration contains 4 triggers, 4 actions, and 4 searches:
 
 ## Triggers
 
-Using the Kentico Kontent integration, you only need to configure the Zap in Zapier. The creation of the webhook in Kontent is handled automatically by the integration; the webhook will be created when you turn on the Zap, and deleted when you turn it off.
+Using the Kontent.ai integration, you only need to configure the Zap in Zapier. The creation of the webhook in Kontent.ai is handled automatically by the integration; the webhook will be created when you turn on the Zap, and deleted when you turn it off.
 
-The output of a trigger in Zapier is exactly what [Kontent's webhook](https://kontent.ai/learn/reference/webhooks-reference) POSTs to Zapier. It will looks something like this:
+The output of a trigger in Zapier is exactly what [Kontent.ai's webhook](https://kontent.ai/learn/reference/webhooks-reference) POSTs to Zapier. It will look something like this:
 
 ```js
-{
-    data: {
-        items: [
-            {
-                item: {
-                    id: '42c21e82-0772-4d79-a6b3-c916e51b24ff'
-                },
-                language: {
-                    id: '00000000-0000-0000-0000-000000000000'
-                }
-            }
-        ]
-    },
-    message: {
-        id: 'a268da50-b3c5-4d09-9b36-6587c8dea500',
-        project_id: '11a3492b-cd32-0054-51d2-8234ec4244a6',
-        type: 'content_item_variant',
-        operation: 'restore',
-        api_name: 'content_management',
-        created_timestamp: '2019-07-18T10:52:33.1059256Z',
-        webhook_url: 'https://hooks.zapier.com/hooks/standard/47991d003732'
-    }
+return {
```

Comment:
I meant like removing the return statement

```suggestion
{
```

Reply:
I added the return statement, because otherwise it is not a valid js (at least according to the markdown extension of my ide. But it doesn't really matter. :)